apps/org.videolan.VLC/eos-vlc-helper.appdata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>vlc.desktop</id>
```

Comment:
Yes, it should be, good catch.

As for the naming, I think that should be fine, as `eos-vlc` will not clash with anything else.